Contrib/Fastcluster/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.pyc
```

Comment:
I don't think this .gitignore file needs to be (or really should be) part of the PR

Reply:
Dear Greg,
Thank you for your advice. I removed .gitignore file.